make/modules/jdk.localedata/Gensrc.gmk

```diff
@@ -36,6 +36,7 @@ TARGETS += $(GENSRC_LOCALEDATA)
 CLDR_DATA_DIR := $(TOPDIR)/make/data/cldr/common
 GENSRC_DIR := $(SUPPORT_OUTPUTDIR)/gensrc/jdk.localedata
 CLDR_GEN_DONE := $(GENSRC_DIR)/_cldr-gensrc.marker
+TZ_DATA_DIR := $(MODULE_SRC)/../java.base/share/data/tzdata
```

Comment:
This seems like a bit of abuse of `$(MODULE_SRC)`. I thought we had a macro to get the proper source path for another module, but if we do, I cannot find it now.

If you feel up for it, I'd encourage you to change this to:

```suggestion
TZ_DATA_DIR := $(call FindModuleSrc, java.base)/share/data/tzdata
```

and add 
```
################################################################################
# Find source dir for module
# Param 1 - module name
FindModuleSrc = \
    $(TOPDIR)/src/$(strip $1)
```
at a suitable place in `Utils.gmk`.

Otherwise, just replace it with:

```
TZ_DATA_DIR :=  $(TOPDIR)/src/java.base/share/data/tzdata
```

Reply:
Thanks, Magnus. In fact, I was wondering if I should move back tzdata into make as it is shared like cldr, but it would revert the work you did before, so I ended up traversing the directory structure. Fixed with the latter suggestion.